debezium-core/src/main/java/io/debezium/relational/ddl/DataTypeBuilder.java

```diff
@@ -22,7 +22,8 @@ public class DataTypeBuilder {
     private int scale = -1;
     private int arrayDimsLength = 0;
     private final int[] arrayDims = new int[40];
-    private static final Pattern SIGNED_UNSIGNED_PATTERN = Pattern.compile("(.*)SIGNED UNSIGNED(.*)", Pattern.CASE_INSENSITIVE);
+    private static final Pattern SIGNED_UNSIGNED_ZEROFILL_PATTERN = Pattern
+            .compile("(.*)\\s+(SIGNED UNSIGNED ZEROFILL|SIGNED UNSIGNED|SIGNED ZEROFILL)", Pattern.CASE_INSENSITIVE);
```

Comment:
Is this safe? Is the orderd of keywords guranteed?

Reply:
Yes,  the method `[setSuffixTokens](https://github.com/debezium/debezium/blob/d5963f16010092c623ba437dd98d1b6fa22a66d6/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/MySqlAntlrDdlParser.java#L136)` of `DataTypeEntry` will keep the ordered of keywords.